### PR TITLE
it's -> its

### PR DESCRIPTION
**Pull request by rloyz**
Pronouns don't have possessive apostrophes. It's is short for it is.

### Description

 fixed bad grsammar

### Motivation

 Bad grammar is cringeworthy in published code.

### CODE REVIEW

**Review by teoli2003 — APPROVED** [review]
👍

**Comment by teoli2003** [comment]
Thanks a lot!

This PR looks good to me.

Congratulations upon your first merged commit here! Welcome aboard! ✌️

File: files/en-us/web/javascript/guide/iterators_and_generators/index.md [hunk 1]
```diff
@@ -120,7 +120,7 @@ console.log(it[Symbol.iterator]() === it); // true
 // which has the @@iterator method return the it (itself),
 // and consequently, the it object can iterate only _once_.
 
-// If we change it's @@iterator method to a function/generator
+// If we change its @@iterator method to a function/generator
 // which returns a new iterator/generator object, (it)
 // can iterate many times
```
Comment (Josh-Cena) [inline files/en-us/web/javascript/guide/iterators_and_generators/index.md:125]:
```suggestion
```suggestion
// If we change the @@iterator method of `it` to a function/generator
// which returns a new iterator/generator object, `it`
// can iterate many times
```

Does that make more sense?

**Review by Josh-Cena — COMMENTED** [review]
Merged too fast😅 This sentence is actually correct, although very confusing.

File: files/en-us/web/javascript/guide/iterators_and_generators/index.md (hunk 1 above)
Comment (teoli2003) [reply files/en-us/web/javascript/guide/iterators_and_generators/index.md:125]: Ha! Yes it was confusing.

As I merged too quickly, I'll do the follow-up with that sentence.

**Comment by teoli2003** [comment]
> I wish English doesn't have `'s` and always uses prepositions like French "de"...

Blame the Viking invasion (that's a Germanic grammar rule inherited from Old Norse)

**Comment by Josh-Cena** [comment]
Blame the Norman conquest for happening too late to change English grammar!

**Comment by rloyz** [comment]
I like this change. It clearly separates the iterator's name "it" from the
English word "it" which is how the original would have been naturally read.

**Pull request merged**